_includes/_results-key.md

```diff
@@ -0,0 +1,9 @@
+
+| Interview 1 Assessment Rating | Interview 2 Assessment Rating | Overall Outcome | Final Score| Category |
+| :--- | :--- | :--- | :--- |
+| 1 | 1 | Applicant Meets Qualifications | 70 | Qualified |
+| 1 | 2 | Applicant Meets Qualifications | 85 | Well Qualified |
+| 2 | 1 | Applicant Meets Qualifications | 85 | Well Qualified |
+| 2 | 2 | Applicant Meets Qualifications | 100 | Best Qualified |
+| 1 or 2 | 0 | Applicant Does Not Meet Qualifications |
```

Comment:
why does it say if you get a 1 or 2 you dont meet quals? those are passing scores now, no?